tools/cli/go-whisk-cli/commands/api.go

```diff
@@ -445,8 +453,11 @@ var apiListCmd = &cobra.Command{
  * Takes an API object (containing one more more single basepath/relpath/operation triplets)
  * and some filtering configuration.  For each API endpoint matching the filtering criteria, display
  * each endpoint's configuration - one line per configuration property (action name, verb, api name, api gw url)
+ * Parses and initializes values for ApiFilteredRow struct
  */
-func printFilteredListApi(resultApi *whisk.RetApi, api *whisk.ApiOptions) {
+func parseFilteredList(resultApi *whisk.RetApi, api *whisk.ApiOptions,flagType string) []whisk.ApiFilteredList {
```

Comment:
I think the argument label of `flagType` is somewhat generic and should reflect it's intention (i.e. sortAlg) - with constant values defined as the supported api sort algorithms.